src/interfaces/WidgetApiAction.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 The Matrix.org Foundation C.I.C.
+ * Copyright 2020 - 2024 The Matrix.org Foundation C.I.C.
```

Comment:
Do we want to touch this files copyright year if there is no change to it?

Reply:
Whoops, this is a leftover.  Thanks!